### PR TITLE
disable shields for any protocol other than http or https

### PR DESCRIPTION
**Pull request by cezaraugusto**
fix brave/brave-browser#169

Test Plan:

covered by automated tests

Manual Test Plan:

Any website using protocols other than `http://`/`https://` such as `chrome://` or `file://` should have shields disabled without the option to enable it

### CODE REVIEW

File: app/helpers/urlUtils.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const isHttpOrHttps = (url?: string) => {
+  if (!url) {
+    return false
+  }
+  return /^http(?=s:|:)/.test(url)
```
Comment (bbondy) [inline app/helpers/urlUtils.ts]: Slightly easier to read way to match here:
`/^https?:/`

Comment (bbondy) [reply app/helpers/urlUtils.ts]: Actually `/^https?:/i` so you can test with `HTTPS://`. Please add a test case for that one too.

Comment (cezaraugusto) [reply app/helpers/urlUtils.ts]: done 👍

**Comment by cezaraugusto** [comment]
ok ready for re-check

**Pull request merged**